config/module.config.php

```diff
@@ -1,49 +1,55 @@
 <?php
 /**
  * @license   http://opensource.org/licenses/BSD-3-Clause BSD-3-Clause
- * @copyright Copyright (c) 2014 Zend Technologies USA Inc. (http://www.zend.com)
+ * @copyright Copyright (c) 2014-2016 Zend Technologies USA Inc. (http://www.zend.com)
  */
 
-return array(
-    'asset_manager' => array(
-        'resolver_configs' => array(
-            'paths' => array(
+namespace ZF\Apigility;
+
+use Zend\Db\Adapter\AdapterAbstractServiceFactory as DbAdapterAbstractServiceFactory;
+use Zend\ServiceManager\Factory\InvokableFactory;
+
+return [
+    'asset_manager' => [
+        'resolver_configs' => [
+            'paths' => [
                 __DIR__ . '/../asset',
-            ),
-        ),
-    ),
-    'router' => array(
-        'routes' => array(
-            'zf-apigility' => array(
-                'type'  => 'Zend\Mvc\Router\Http\Literal',
-                'options' => array(
+            ],
+        ],
+    ],
+    'router' => [
+        'routes' => [
+            'zf-apigility' => [
+                'type'  => 'literal',
```

Comment:
Using `::class` notation here is a no-go at this time, due to the fact that we cannot know which of zend-mvc v2 or zend-router v{any} are present. As such, I updated to use the "short name" aliases that are present in both.

I'll fix the space issues shortly.